### PR TITLE
Ports server to 514, updates atmos to TauCeti state. 

### PR DESCRIPTION
**Pull request by drexample**
## About The Pull Request

Guys holy hell Eris at 45+ FPS is so smooth I can't fucking handle it 

## Why It's Good For The Game

https://cdn.discordapp.com/attachments/265411548963405824/891110320087629864/2021-09-25_02-52-53.mp4

Atmos lags less
Atmos machinery lags less
Venry verny good fps looks dope

## Changelog
:cl:
refactor: Made server 514 ready and replaced ticklag with FPS in a few places
refactor: Replaced our air subsystem with TauCeti's air.
/:cl:

### CODE REVIEW

File: code/__DEFINES/tick.dm
Comment (MLGTASTICa) [inline code/__DEFINES/tick.dm:28]: Line 7 also has this defined

Comment (MLGTASTICa) [reply code/__DEFINES/tick.dm:28]: TICK_LIMIT_MC and TICK_LIMIT_MC_INIT_DEFAULT

File: code/game/world.dm
Comment (Iglpigl) [inline code/game/world.dm:397]: No newline.

File: config/example/config.txt
Comment (hyperioo) [inline config/example/config.txt:212]:
```suggestion
## Defines the FPS for the world. 30 is the normal one, 45+ is smoother.
```

File: html/add2changelog.html
Comment (hyperioo) [inline html/add2changelog.html:77]:
```suggestion
</html>

```

File: html/templates/footer.html
Comment (hyperioo) [inline html/templates/footer.html:5]: These are not really up to date

File: html/templates/header.html
Comment (hyperioo) [inline html/templates/header.html:41]: Same here, not up to date

File: html/add2changelog.html
Comment (drexample) [reply html/add2changelog.html:77]: I have no clue what keeps breaking those on my side

**Comment by drexample** [comment]
Whitespace changes exist to clog PRs, god why

**Review by hyperioo — APPROVED** [review]
Seems good, if it has been tested in local and does not blow up the server.

**Review by TheShown911 — APPROVED** [review]
Looks like it worked well on the test server. Don't merge without head dev approval, may need some work with the host.

**Pull request merged**
